src/components/App/App.js

```diff
@@ -7,10 +8,25 @@ import { Footer } from "../Footer/Footer";
 import SearchPage from "../SearchPage/SearchPage";
 import StoresPage from "../StoresPage/StoresPage";
 import CartPage from "../CartPage/CartPage";
+import { GET_USER } from "../../hooks/getQueries";
+import { useLazyQuery, useMutation } from "@apollo/client";
 
 const App = () => {
+  const [user, setUser] = useState(0);
+  const [getUser, { loading, error }] = useLazyQuery(GET_USER, {
```

Comment:
I would update this from 'useLazyQuery' to 'useQuery' which causes the 'get' to happen load upon page/app load, then you could remove the useEffect block (lines 20-26) below entirely. The onCompleted on line 17 would then set data to setUser how you have it written currently, no changes would be needed there.